### PR TITLE
Version Increment to 1.0 and changelog updates

### PR DESCRIPTION
**Pull request by Colin-Stone**
Signed-off-by: Colin Stone <30794003+Colin-Stone@users.noreply.github.com>

Please update and enhance as appropriate. Will need to be merged after #329

### CODE REVIEW

**Comment by zFernand0** [comment]
It's looking good! :) 
I think we should add something related to https://github.com/zowe/vscode-extension-for-zowe/pull/353 as well

**Comment by Colin-Stone** [comment]
> It's looking good! :)
> I think we should add something related to #353 as well

That was a little `left field` to me as I didn't know it was going to go in

**Comment by zFernand0** [comment]
Thanks for adding a few lines about the newest fix/enhancement :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+# 1.0.0
```
Comment (IgorCATech) [inline CHANGELOG.md]: Lint issues with the heading. Should use ## instead of #

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+
+- VSCode centric Connection settings. . Thanks @crawr, @jellypuno
+    - Credential prompting in profiles and favorite . Thanks @crawr, @jellypuno
```
Comment (IgorCATech) [inline CHANGELOG.md]: Indentation issues - used 4 instead of 2 'spaces'

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+- Dataset and Dataset member copy and renaming function. Thanks @CForrest97
+- Theia support including documentation 
+- Improvements to Save and Safe Save functionality. Thanks Alexandru-Dumitru
+- Reliability and Resiliance updates:
+    - for default profiles
```
Comment (IgorCATech) [inline CHANGELOG.md]: Same as in the line 8

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+# 1.0.0
+
+- VSCode centric Connection settings. . Thanks @crawr, @jellypuno
+    - Credential prompting in profiles and favorite . Thanks @crawr, @jellypuno
+- Dataset and Dataset member copy and renaming function. Thanks @CForrest97
+- Theia support including documentation
```
Comment (IgorCATech) [inline CHANGELOG.md]: Unnecessary trailing space

Comment (Colin-Stone) [reply CHANGELOG.md]: Whoops sorry guys. Should have run the linter.

**Comment by zFernand0** [comment]
:smile:
As a reminder:
Please do not merge until everything related to the GA 1.0 milestone is merged

**Comment by IgorCATech** [comment]
@zFernand0 Are there many dependencies? I'd like to know when it's possible to publish Zowe docs.

**Comment by zFernand0** [comment]
> @zFernand0 Are there many dependencies? I'd like to know when it's possible to publish Zowe docs.

We are good to go :)

**Pull request merged**